pkg/api/models/kluster_print.go

```diff
@@ -31,7 +31,10 @@ func (k Kluster) Print(format printers.PrintFormat, options printers.PrintOption
 func (k Kluster) printHuman(options printers.PrintOptions) {
 	fmt.Println("Cluster name: ", k.Name)
 	fmt.Println("Cluster state: ", k.Status.Phase)
-	fmt.Println("Cluster CIDR: ", k.Spec.ClusterCIDR)
+	if k.Spec.ClusterCIDR != nil && *k.Spec.ClusterCIDR != "" {
+		fmt.Println("Cluster CIDR: ", *k.Spec.ClusterCIDR)
+	}
+
```

Comment:
If i read this correct the clusterCIDR will just be missing from the output.
Spotting a missing line in debug is way harder than printing: *** UNSET *** or something similar.

Reply:
Ok I think I've addressed this. I now print the cluster CIDR if its not `nil`. Which should not happen at all because. I now also keep it as an empty string in storage (CRD). This means the only time this should be actually nil is during cluster creation.